Frameworks/Foundation/NSURLConnection.mm

```diff
@@ -110,18 +109,15 @@ @implementation NSURLConnection {
     StrongId<NSURLResponse> _response;
     NSURLCacheStoragePolicy _storagePolicy;
     StrongId<NSURLProtocol> _protocol;
+
+    bool _started;
```

Comment:
(is this superior to property autosynthesis? it looks like there's already a property for this ivar, and the ivar will be automatically generated just like it was before this change)